feat(content): Hai culture conversation for acorn bread (#9738)
endless-sky/endless-sky: data/hai/hai culture conversations.txt

				decline
			label translator
			`	You find a movie theater showing Likeet's film, but your translator is of little help here. The movie is sparse on dialog, and the few lines spoken in the movie seem to be cryptic references to Hai literature and philosophy. The most you are able to make out is that the movie is about an unseen monster inhabiting the station that slowly drives those it meets insane. It seems to be a hit with the Hai audience, though, and you leave the theater feeling distinctly unnerved.`

mission "Sick of Acorn Bread"
	minor

> 	source
> 		government "Hai"
> 		not attributes "station"

Reviewer: Attribute it to areas with a high Human presence
```suggestion
	source
		government "Hai"
		not attributes "station"
		attributes "hai: human presence" "human tourism"
```

	to offer
		credits > 50
		random < 1
	on offer
		conversation

> 			`With some time to kill while your ship is being serviced by the Hai dockhands, you enter the spaceport searching for a bite to eat. Restaurants advertise their menus in bright, attention-grabbing colors on their windows; amongst the selection, you spot one that promises "authentic Hai cuisine," along with some large text printed in the Hai language which you don't understand. Next to it, you see a human street food vendor offering snacks and ready-made meals.`

Reviewer: The translator you get later includes text to speech, so you'll either want to have a branch here on the text if you've got the translator, or remove the specific statement that you don't understand the language.

			choice
				`"I'll have today's special."`
				`"I'd like some soup, please."`
			`	"You're in luck," she says. "Today only, each meal is served with a free side of acorn bread." She scampers off to place and retrieve your order.`
			`	The chubby Hai next to you grunts again - this time with clear frustration - and turns to speak. "Acorn bread! That's all they serve here anymore. Ever since you humans arrived in our space, that's all we get to eat. Traditional Hai breadmaking has become a lost art."`
			choice
				`"You mean there's more than just acorn bread?"`
					goto breads
				`"I'm sorry. I didn't mean to offend your culture."`

> 			`	The look on his face softens. "I should be the one to apologize. It is not your fault that Hai culture has somewhat eroded in this respect. Humans have also brought new life to many parts of Hai society. It is only a pity that we lack the many types of bread that we had in years past."`

Reviewer: It's probably not correct to make the claim that Hai culture has been eroded, but probably should say something to the effect of spaceports and other high-traffic areas being dominated by human-friendly commercialised foods.
Having to get all the way into town and away from the spaceport to get a traditional snack would be an annoying inconvenience to regular travellers.

			`	"You mean there was more than just acorn bread?" you ask.`
			label breads

> 			`	"Oh yes. Acorn bread was a rare sight in Hai cuisine. It's too sweet, you see." He shifts his weight on the stool. "I'm old enough to remember hayipp bread, yur ki bread - you name it. Hayipp is a grain, similar to your human 'corn,' which creates a wholesome ruddy brown flour when milled. The bread was hearty, yet exquisite." You note the wistful look on his face. "Yur ki is a milk, sourced from the livestock we raise. You may have seen some on Newhome; I'm told they resemble your human 'sheep.' Bread made from yur ki is rich and decadent. It is among my favorite foods to have for breakfast." He pauses for a moment and looks a bit downcast. "Since humans have joined us on our planets, all they want is acorn bread. The high demand has displaced all of the traditional breads that I grew up with. I miss them."`

Reviewer: In line with the above comment - the expected difficulty would be that if you lived near a spaceport for work or business reasons, it might be annoyingly difficult to source more traditional breads.

				decline
			label restaurant
			`	A Hai restaurant is sure to provide a more unique experience. The door jingles a short melody as you enter the establishment, and a middle-aged female Hai shouts something at you in the Hai language.`
			branch translator
				has "language: Hai"
			`	You freeze for a second, as it is unclear what the Hai wants from you.`
				goto yourorder
			label translator
			`	Your translator chirps dutifully, "Have a [seat, rest] anywhere you like! I'll be right with you."`
			label yourorder
			`	Only a short moment later, the same Hai hostess peers over at you and chitters in what you assume is embarrassment. "I'm so sorry. I didn't realize I would have the pleasure of serving a human today. Please, have a seat anywhere at the bar. I will come to take your order shortly."`

> 			`	You take a look around and note that the restaurant is actually quite crowded. There are Hai dressed in fancy tuxedos, fashionable clothing, casual dress, and even some wearing peculiar overalls with holes in the backs for their tails; there don't seem to be any other humans. With only a few seats available, you select one on the end, next to a rather fat Hai wearing green overalls and an orange straw hat. He grunts noncommittally as you sit down on a stool clearly designed for an organism with a tail. Before long, the hostess approaches and asks for your order.`

Reviewer 2: ```suggestion
			`	You take a look around and note that the restaurant is actually quite crowded. There are Hai dressed in fancy tuxedos, fashionable clothing, casual dress, and even some wearing peculiar overalls with holes in the backs for their tails; there don't seem to be any other humans. With only a few seats available, you select one on the end, next to a rather large Hai wearing green overalls and an orange straw hat. He grunts noncommittally as you sit down on a stool clearly designed for an organism with a tail. Before long, the hostess approaches and asks for your order.`
```
Just... nicer, I guess. I don't think specifying him as `fat` or `chubby` particularly adds anything to his character.

			choice
				`"I'll have today's special."`
				`"I'd like some soup, please."`
			`	"You're in luck," she says. "Today only, each meal is served with a free side of acorn bread." She scampers off to place and retrieve your order.`

> 			`	The chubby Hai next to you grunts again - this time with clear frustration - and turns to speak. "Acorn bread! That's all they serve here anymore. Ever since you humans arrived in our space, that's all we get to eat. Traditional Hai breadmaking has become a lost art."`

Reviewer 2: ```suggestion
			`	The Hai next to you grunts again - this time with clear frustration - and turns to speak. "Acorn bread! That's all they serve here anymore. Ever since you humans arrived in our space, that's all we get to eat. Traditional Hai breadmaking has become a lost art."`
```

			choice

> 				`"You mean there's more than just acorn bread?"`

Reviewer 3: Is there anything that would suggest to the player that acorn bread is the only type of bread the Hai produce?

Reviewer: The implication of the line above it is that acorn bread is so comparatively popular with humans, that touristy places like spaceport thoroughfares almost exclusively stock just that.
So if you were a human pilot who rarely ventured beyond the spaceport, that would suggest it to the player _character_. For this option though, the implication may need to be strengthened for the _player_.

					goto breads
				`"I'm sorry. I didn't mean to offend your culture."`

> 			`	The look on his face softens. "I should be the one to apologize. It is not your fault that our spaceports' menus have changed in this respect. Humans have also brought new life to many parts of Hai society. It is only a pity that we have to travel far to find the types of bread that we had in years past."`

Reviewer 2: ```suggestion
			`	The look on his face softens. "I should be the one to apologize. It's not your fault that our spaceports' menus have changed in this respect. Humans have also brought new life to many parts of Hai society. It is only a pity that we have to travel far to find the types of bread that we had in years past."`
```

			`	"You mean there was more than just acorn bread?" you ask.`
			label breads

> 			`	"Oh yes. Acorn bread was a rare sight in Hai cuisine. It's too sweet, you see." He shifts his weight on the stool. "I'm old enough to remember hayipp bread, yur ki bread - you name it. Hayipp is a grain, similar to your human 'corn,' which creates a wholesome ruddy brown flour when milled. The bread was hearty, yet exquisite." You note the wistful look on his face. "Yur ki is a milk, sourced from the livestock we raise. You may have seen some on Newhome; I'm told they resemble your human 'sheep.' Bread made from yur ki is rich and decadent. It is among my favorite foods to have for breakfast." He pauses for a moment and looks a bit downcast. "Since humans have joined us on our planets, all they want is acorn bread. The high demand has displaced all of the traditional breads that I grew up with from high traffic areas such as this spaceport restaurant. I miss having them easily available."`

Reviewer 2: ```suggestion
			`	"Oh yes. Acorn bread was a rare sight in Hai cuisine. It's too sweet, you see." He shifts his weight on the stool. "I'm old enough to remember hayipp bread, yur ki bread - you name it. Hayipp is a grain, similar to your human 'corn,' which creates a wholesome ruddy brown flour when milled. The bread was hearty, yet exquisite." You note the wistful look on his face. "Yur ki is a milk, sourced from the livestock we raise. You may have seen some on Newhome; I'm told your kind has a similar animal called 'sheep.' Bread made from yur ki is rich and decadent. It is among my favorite foods to have for breakfast." He pauses for a moment and looks a bit downcast. "Since humans have joined us on our planets, all they want is acorn bread. The high demand has displaced all of the traditional breads that I grew up with from high traffic areas such as this spaceport restaurant. I miss having them easily available."`
```

			action
				payment -50

> 			`	As if on cue, the hostess arrives with both of your meals. The Hai next to you, whose name you still do not know, sighs as he sees his free acorn bread, but he eats it nonetheless. You each finish your meals in silence and leave.`

Reviewer 4: ```suggestion
			`	As if on cue, the hostess arrives with both of your meals. The Hai next to you, whose name you still do not know, sighs as he sees his free acorn bread; he eats it nonetheless. You each finish your meals in silence and leave.`
```

Author: I think this sounds better without the semicolon. `but` is doing its job in this sentence.

Reviewer 4: The sentence sounds a bit run-on without the semi-colon, imo. I'm not super gung-ho about the change anyways, so up to you.

				decline
			label translator
			`	You find a movie theater showing Likeet's film, but your translator is of little help here. The movie is sparse on dialog, and the few lines spoken in the movie seem to be cryptic references to Hai literature and philosophy. The most you are able to make out is that the movie is about an unseen monster inhabiting the station that slowly drives those it meets insane. It seems to be a hit with the Hai audience, though, and you leave the theater feeling distinctly unnerved.`

mission "Sick of Acorn Bread"
	minor
	source
		government "Hai"
		not attributes "station"
	to offer
		credits > 50
		random < 1
	on offer
		conversation

> 			`With some time to kill before your next mission, you enter the spaceport searching for a bite to eat. Restaurants advertise their menus in bright colors on their windows, each competing for your attention. You spot one that promises "authentic Hai cuisine," along with some large text printed in the Hai language which you don't understand. Next to it, you see a human street food vendor offering snacks and ready-made meals.`

Reviewer 5: What if you have the translator? Does it just not work on written text? (It could, I'm not sure)

> 			`With some time to kill while your ship is being serviced by the Hai dockhands, you enter the spaceport searching for a bite to eat. Restaurants advertise their menus in bright, attention-grabbing colors on their windows. Amongst the selection, you spot one that promises "authentic Hai cuisine," along with some large text printed in the Hai language which you don't understand. Next to it, you see a human street food vendor offering snacks and ready-made meals.`

Reviewer 6: I think I prefer the way I wrote it. Not only does it feel a bit stunted with that full stop (period), I imagine you notice the wording as you look at the windows, and that full stop gets in the way as if noticing it was an afterthought.

Also, I know it's a long sentence but whatever.
```suggestion
			`With some time to kill while your ship is being serviced by the Hai dockhands, you enter the spaceport searching for a bite to eat. Restaurants advertise their menus in bright, attention-grabbing colors on their windows; amongst the selection, you spot one that promises "authentic Hai cuisine," along with some large text printed in the Hai language which you don't understand. Next to it, you see a human street food vendor offering snacks and ready-made meals.`
```

			choice
				`	(Enter the restaurant.)`
					goto restaurant
				`	(Choose the human food vendor instead.)`

> 					goto humanfood
> 				`	(On second thought, I'm not that hungry.)`
> 					decline
> 			label humanfood

Reviewer 6: ```suggestion
				`	(On second thought, I'm not that hungry.)`
					decline
```
These aren't strictly necessary, may as well save a few lines.

			`	You take a look around and note that the restaurant is actually quite crowded. There are Hai dressed in fancy tuxedos, fashionable clothing, casual dress, and even some wearing peculiar overalls with holes in the backs for their tails; there don't seem to be any other humans. With only a few seats available, you select one on the end, next to a rather fat Hai wearing green overalls and an orange straw hat. He grunts noncommittally as you sit down on a stool clearly designed for an organism with a tail. Before long, the hostess approaches and asks for your order.`
			choice
				`"I'll have today's special."`
				`"I'd like some soup, please."`
			`	"You're in luck," she says. "Today only, each meal is served with a free side of acorn bread." She scampers off to place and retrieve your order.`
			`	The chubby Hai next to you grunts again - this time with clear frustration - and turns to speak. "Acorn bread! That's all they serve here anymore. Ever since you humans arrived in our space, that's all we get to eat. Traditional Hai breadmaking has become a lost art."`
			choice

> 				`"You mean there is more than just acorn bread?"`

Reviewer 6: ```suggestion
				`"You mean there's more than just acorn bread?"`
```

					goto breads
				`"I'm sorry. I didn't mean to offend your culture."`
			`	The look on his face softens. "I should be the one to apologize. It is not your fault that Hai culture has somewhat eroded in this respect. Humans have also brought new life to many parts of Hai society. It is only a pity that we lack the many types of bread that we had in years past."`
			`	"You mean there was more than just acorn bread?" you ask.`
			label breads

> 			`	"Oh yes. Acorn bread was a rare sight in Hai cuisine. It's too sweet, you see." He shifts his weight on the stool. "I'm old enough to remember hayipp bread, yur ki bread - you name it. Hayipp is a grain, similar to your human 'corn,' which creates a wholesome ruddy brown flour when milled. The bread was hearty, yet exquisite." You note the wistful look on his face. "Yur Ki is a milk, sourced from the livestock we raise - you may have seen some on Newhome; I'm told they resemble your human 'sheep.' Bread made from yur ki is rich and decadent. It is among my favorite foods to have for breakfast." He pauses for a moment and looks a bit downcast. "Since humans have joined us on our planets, all they want is acorn bread. The high demand has displaced all of the traditional breads that I grew up with. I miss them."`

Reviewer 6: ```suggestion
			`	"Oh yes. Acorn bread was a rare sight in Hai cuisine. It's too sweet, you see." He shifts his weight on the stool. "I'm old enough to remember hayipp bread, yur ki bread - you name it. Hayipp is a grain, similar to your human 'corn,' which creates a wholesome ruddy brown flour when milled. The bread was hearty, yet exquisite." You note the wistful look on his face. "Yur ki is a milk, sourced from the livestock we raise. You may have seen some on Newhome; I'm told they resemble your human 'sheep.' Bread made from yur ki is rich and decadent. It is among my favorite foods to have for breakfast." He pauses for a moment and looks a bit downcast. "Since humans have joined us on our planets, all they want is acorn bread. The high demand has displaced all of the traditional breads that I grew up with. I miss them."`
```
